OrbitLinuxTracing/PerfEventReaders.cpp

```diff
@@ -52,6 +52,25 @@ std::unique_ptr<SamplePerfEvent> ConsumeSamplePerfEvent(
   return event;
 }
 
+std::unique_ptr<SampleFPPerfEvent> ConsumeSampleFPPerfEvent(
```

Comment:
Rename to ConsumeSampleCallchainPerfEvent.

Reply:
Done.